diff --git a/packages/api/pyproject.toml b/packages/api/pyproject.toml
index dcf65f1..560d2cd 100644
--- a/packages/api/pyproject.toml
+++ b/packages/api/pyproject.toml
@@ -4,25 +4,3 @@ edition = "2018"
 [build-system]
 requires = ["maturin>=0.14,<0.15"]
 build-backend = "maturin"
-
-[tool.maturin.metadata]
-category = ["Programming Language :: Python",
-    "Programming Language :: Python :: 3",
-    "Programming Language :: Python :: 3.7",
-    "Programming Language :: Python :: 3.8",
-    "Programming Language :: Python :: 3.9",
-    "Programming Language :: Python :: 3.10",
-    "Programming Language :: Python :: 3.11",
-    "Programming Language :: Python :: Implementation :: CPython",
-    "Programming Language :: Rust",
-    "Topic :: Software Development",
-    "Topic :: Software Development :: Compilers",
-    "Topic :: Software Development :: Interpreters",
-    "License :: OSI Approved :: MIT License",
-    "Operating System :: OS Independent"]
-
-
-[tool.maturin.project-url]
-"Source Code" = "https://github.com/wasmerio/wasmer-python/"
-"Bug Tracker" = "https://github.com/wasmerio/wasmer-python/issues"
-"Documentation" = "https://github.com/wasmerio/wasmer-python/"
diff --git a/packages/compiler-cranelift/pyproject.toml b/packages/compiler-cranelift/pyproject.toml
index dcf65f1..560d2cd 100644
--- a/packages/compiler-cranelift/pyproject.toml
+++ b/packages/compiler-cranelift/pyproject.toml
@@ -4,25 +4,3 @@ edition = "2018"
 [build-system]
 requires = ["maturin>=0.14,<0.15"]
 build-backend = "maturin"
-
-[tool.maturin.metadata]
-category = ["Programming Language :: Python",
-    "Programming Language :: Python :: 3",
-    "Programming Language :: Python :: 3.7",
-    "Programming Language :: Python :: 3.8",
-    "Programming Language :: Python :: 3.9",
-    "Programming Language :: Python :: 3.10",
-    "Programming Language :: Python :: 3.11",
-    "Programming Language :: Python :: Implementation :: CPython",
-    "Programming Language :: Rust",
-    "Topic :: Software Development",
-    "Topic :: Software Development :: Compilers",
-    "Topic :: Software Development :: Interpreters",
-    "License :: OSI Approved :: MIT License",
-    "Operating System :: OS Independent"]
-
-
-[tool.maturin.project-url]
-"Source Code" = "https://github.com/wasmerio/wasmer-python/"
-"Bug Tracker" = "https://github.com/wasmerio/wasmer-python/issues"
-"Documentation" = "https://github.com/wasmerio/wasmer-python/"
diff --git a/packages/compiler-llvm/pyproject.toml b/packages/compiler-llvm/pyproject.toml
index dcf65f1..560d2cd 100644
--- a/packages/compiler-llvm/pyproject.toml
+++ b/packages/compiler-llvm/pyproject.toml
@@ -4,25 +4,3 @@ edition = "2018"
 [build-system]
 requires = ["maturin>=0.14,<0.15"]
 build-backend = "maturin"
-
-[tool.maturin.metadata]
-category = ["Programming Language :: Python",
-    "Programming Language :: Python :: 3",
-    "Programming Language :: Python :: 3.7",
-    "Programming Language :: Python :: 3.8",
-    "Programming Language :: Python :: 3.9",
-    "Programming Language :: Python :: 3.10",
-    "Programming Language :: Python :: 3.11",
-    "Programming Language :: Python :: Implementation :: CPython",
-    "Programming Language :: Rust",
-    "Topic :: Software Development",
-    "Topic :: Software Development :: Compilers",
-    "Topic :: Software Development :: Interpreters",
-    "License :: OSI Approved :: MIT License",
-    "Operating System :: OS Independent"]
-
-
-[tool.maturin.project-url]
-"Source Code" = "https://github.com/wasmerio/wasmer-python/"
-"Bug Tracker" = "https://github.com/wasmerio/wasmer-python/issues"
-"Documentation" = "https://github.com/wasmerio/wasmer-python/"
diff --git a/packages/compiler-singlepass/pyproject.toml b/packages/compiler-singlepass/pyproject.toml
index dcf65f1..560d2cd 100644
--- a/packages/compiler-singlepass/pyproject.toml
+++ b/packages/compiler-singlepass/pyproject.toml
@@ -4,25 +4,3 @@ edition = "2018"
 [build-system]
 requires = ["maturin>=0.14,<0.15"]
 build-backend = "maturin"
-
-[tool.maturin.metadata]
-category = ["Programming Language :: Python",
-    "Programming Language :: Python :: 3",
-    "Programming Language :: Python :: 3.7",
-    "Programming Language :: Python :: 3.8",
-    "Programming Language :: Python :: 3.9",
-    "Programming Language :: Python :: 3.10",
-    "Programming Language :: Python :: 3.11",
-    "Programming Language :: Python :: Implementation :: CPython",
-    "Programming Language :: Rust",
-    "Topic :: Software Development",
-    "Topic :: Software Development :: Compilers",
-    "Topic :: Software Development :: Interpreters",
-    "License :: OSI Approved :: MIT License",
-    "Operating System :: OS Independent"]
-
-
-[tool.maturin.project-url]
-"Source Code" = "https://github.com/wasmerio/wasmer-python/"
-"Bug Tracker" = "https://github.com/wasmerio/wasmer-python/issues"
-"Documentation" = "https://github.com/wasmerio/wasmer-python/"
